### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Anton-Zhytonbaiev**
[DEMO LINK](https://Anton-Zhytonbaiev.github.io/react_movies-list/)

### CODE REVIEW

File: src/components/MovieCard/MovieCard.tsx [hunk 1]
```diff
@@ -1,7 +1,62 @@
...
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  } = movie;
+
+  return (
+    <>
```
Comment (NVyrsta) [inline src/components/MovieCard/MovieCard.tsx:19]: redundant fragment

File: src/components/MoviesList/MoviesList.tsx [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <>
```
Comment (NVyrsta) [inline src/components/MoviesList/MoviesList.tsx:11]: the same here

**Review by NVyrsta — CHANGES_REQUESTED** [review]
Good job 👍
Just delete redundant wrappers

**Review by alexandra-protyanova — APPROVED** [review]
Well done 🔥

**Pull request closed without merging**